### PR TITLE
fix schema to match erd and loaded data

### LINKED ISSUE #50: Amend schema.sql (2)
Theres a couple issues left with the schema.sql before it's suitable for use:

- [ ] `artist.url` should not unique (multiple artists can have the same page, because some pages belong to labels)
- [ ] `track.album_id` should be nullable (many songs do not belong to albums)

### PR DESCRIPTION
**Pull request by e-lemma**
Closes #50 

I've just changed some column constraints in schema.sql to allow for different artists with the same url, and tracks that are not associated with albums.

### CODE REVIEW

File: schema/schema.sql [hunk 1]
```diff
@@ -19,7 +19,7 @@ CREATE TABLE subscriber (
 CREATE TABLE artist (
     artist_id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
     name TEXT NOT NULL,
-    url TEXT UNIQUE NOT NULL
+    url TEXT NOT NULL
```
Comment (alina-101) [inline schema/schema.sql:22]: Why should URL not be unique?

Comment (e-lemma) [reply schema/schema.sql:22]: To accommodate for URLS that different artists may share, e.g. record label pages, solo releases by band members.

Comment (alina-101) [reply schema/schema.sql:22]: Oh okay I understand now, thanks

**Review by alina-101 — APPROVED** [review]
Good spot from looking at the data

**Pull request merged**